### ISSUE TITLE
[BUG Report]: Error

### ISSUE THREAD

**Issue by Pix-10** [MatrixTM/MHDDoS#524]
### Version

2.4 (Default)

### Which section is the problem?

Script Base

### Please complete the following information

```true
- OS: Windows 11 Pro
- Installed Requirements Version 
cloudscraper==1.2.71
certifi==2022.12.7
dnspython==2.2.1
requests==2.27.1
impacket==0.10.0
psutil>=5.9.3
icmplib>=2.1.1
pyasn1==0.4.8
pyroxy @ git+https://github.com/MatrixTM/PyRoxy.git
yarl>=1.9.2 

- Python Version 3.12.1
```

### What happened (Describe the bug) ?

It gave me this: 
D:\MHDDoS\start.py:168: SyntaxWarning: invalid escape sequence '\d'
  IP = compile("(?:\d{1,3}\.){3}\d{1,3}")
D:\MHDDoS\start.py:169: SyntaxWarning: invalid escape sequence '\d'
  protocolRex = compile('"protocol":(\d+)')

### Screenshots

https://imgur.com/a/jJzFpqM

### Relevant log output

```shell
D:\MHDDoS\start.py:168: SyntaxWarning: invalid escape sequence '\d'
  IP = compile("(?:\d{1,3}\.){3}\d{1,3}")
D:\MHDDoS\start.py:169: SyntaxWarning: invalid escape sequence '\d'
  protocolRex = compile('"protocol":(\d+)')
```

### Your code

    IP = compile(r"(?:\d{1,3}\.){3}\d{1,3}")
    protocolRex = compile('"protocol":(\\d+)')

**Comment by Kuchiriel** [comment]
Any news?

**Comment by fbicudo** [comment]
Do we have an answer to this issue?

**Comment by vietjovi** [comment]
Let try with the fix https://github.com/MatrixTM/MHDDoS/pull/591/commits/f8e938481d466eb53a782c0a483142cb9f2af26bI 

There are language changes in Python 3.12 - Reference: https://docs.python.org/3/whatsnew/3.12.html#other-language-changes
> A backslash-character pair that is not a valid escape sequence now generates a [SyntaxWarning](https://docs.python.org/3/library/exceptions.html#SyntaxWarning), instead of [DeprecationWarning](https://docs.python.org/3/library/exceptions.html#DeprecationWarning). For example, re.compile("\d+.\d+") now emits a [SyntaxWarning](https://docs.python.org/3/library/exceptions.html#SyntaxWarning) ("\d" is an invalid escape sequence, use raw strings for regular expression: re.compile(r"\d+.\d+")). In a future Python version, [SyntaxError](https://docs.python.org/3/library/exceptions.html#SyntaxError) will eventually be raised, instead of [SyntaxWarning](https://docs.python.org/3/library/exceptions.html#SyntaxWarning). (Contributed by Victor Stinner in https://github.com/python/cpython/issues/98401.)

I also creted a MR here: https://github.com/MatrixTM/MHDDoS/pull/591

**Comment by Kuchiriel** [comment]
> Let try with this fix [f8e9384](https://github.com/MatrixTM/MHDDoS/commit/f8e938481d466eb53a782c0a483142cb9f2af26b)
> 
> ```
> IP = compile(r"(?:\d{1,3}\.){3}\d{1,3}")
> protocolRex = compile(r'"protocol":(\d+)')
> ```
> 
> There are language changes in Python 3.12 - Reference: https://docs.python.org/3/whatsnew/3.12.html#other-language-changes
> 
> > A backslash-character pair that is not a valid escape sequence now generates a [SyntaxWarning](https://docs.python.org/3/library/exceptions.html#SyntaxWarning), instead of [DeprecationWarning](https://docs.python.org/3/library/exceptions.html#DeprecationWarning). For example, re.compile("\d+.\d+") now emits a [SyntaxWarning](https://docs.python.org/3/library/exceptions.html#SyntaxWarning) ("\d" is an invalid escape sequence, use raw strings for regular expression: re.compile(r"\d+.\d+")). In a future Python version, [SyntaxError](https://docs.python.org/3/library/exceptions.html#SyntaxError) will eventually be raised, instead of [SyntaxWarning](https://docs.python.org/3/library/exceptions.html#SyntaxWarning). (Contributed by Victor Stinner in [python/cpython#98401](https://github.com/python/cpython/issues/98401).)
> 
> I also created a MR here: [#591](https://github.com/MatrixTM/MHDDoS/pull/591)

I submited the same PR for the fix with the same fix and they denied for no reason. I not kidding I lost my job because I did a git pull on the company laptop to solve this. This is so upsetting, FOSS only giving me troubles.

**Comment by Kuchiriel** [comment]
> [@Kuchiriel](https://github.com/Kuchiriel) Then don’t do a git pull on the company laptop.

Very enlightening, thanks for your words of light.

I don't have a laptop anymore neither a company to work.

But your very advanced reasoned advice changed my life forever.

It will be on my grave. ❤️

**Comment by mrzt721010** [comment]
anytime!

**Comment by vietjovi** [comment]
Sorry to hear that @Kuchiriel. But in the future, be careful when installing or using hacking/security tools on your company laptop.

**Comment by Kuchiriel** [comment]
> Sorry to hear that [@Kuchiriel](https://github.com/Kuchiriel). But in the future, be careful when installing or using hacking/security tools on your company laptop.

Thanks for the advice, but I wont be working on a company that fire someone for a git pull. I dont need that, It could be solved with a warning, some other treatment, cause I saw ppl doing worst things, using Tor and etc, and nothing happened. I was targeted because I was automating everything in the IaC DevOps cell I've created and ppl had trouble with it. So they target a simple git pull as the worst crime of all time, the same time ppl is saving their password on .txt files on the servers. It was such a joke. It opened my eyes for how big companies work.
